my_list.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'my_enumerable'
+
+class MyList
```

Comment:
- [ ] MyList class should implement the `#each` method that yields all members of `@list` in succession, kindly make sure to implement this function instead of using the `each` method from ruby
e.g
```ruby
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```